### PR TITLE
Make health check work with new spreadsheet format

### PR DESCRIPTION
**Pull request by MatMoore**
We've scrapped weighting based on monthly views, and added tags.

- When printing the summary, show a breakdown by tag.
- When parsing the input, log any duplicates we run into.

Also:
- Adding calculators doesn't seem to be used
- I don't think testing the output format is valuable: it doesn't change often and it's easier to just run the check

The suggestions check doesn't have tags but could. I've just done the minimum there to make it work.

Example output for `./bin/health_check -j "https://www.gov.uk/api/search.json"`:

```
FAIL,/repaying-your-student-loan,university funding,0,<= 3,benchmarking-student-loan-repayment
FAIL,/repaying-your-student-loan,paying back the student loan,6,<= 3,benchmarking-student-loan-repayment
FAIL,/repaying-your-student-loan,student loan pay back,6,<= 3,benchmarking-student-loan-repayment
FAIL,/repaying-your-student-loan,student loan pay back calculator,8,<= 3,benchmarking-student-loan-repayment
FAIL,/repaying-your-student-loan,how to pay back loan,0,<= 3,benchmarking-student-loan-repayment
INFO,benchmarking score: 0 of 1 succeeded
INFO,benchmarking-bank-holidays score: 10 of 18 succeeded
INFO,benchmarking-calculate-holiday score: 6 of 11 succeeded
INFO,benchmarking-check-uk-visa score: 10 of 10 succeeded
INFO,benchmarking-child-maintenance score: 6 of 8 succeeded
INFO,benchmarking-contact-dvla score: 7 of 15 succeeded
INFO,benchmarking-jobseekers-allowance score: 14 of 14 succeeded
INFO,benchmarking-overseas-passports score: 3 of 10 succeeded
INFO,benchmarking-state-pensions score: 9 of 12 succeeded
INFO,benchmarking-student-loan-repayment score: 8 of 18 succeeded
INFO,Score: 73 of 117 succeeded
```

Trello: https://trello.com/c/5OaJZlEM/88-segment-search-healthcheck-results

### CODE REVIEW

File: test/unit/health_check/check_file_parser_test.rb [hunk 1]
```diff
@@ -8,78 +8,54 @@ def checks(data)
...
     end
 
-    should "accept Monthly searches values containing commas" do
-      data = "Monthly searches,When I search for...,Then I...,see...,in the top ... results
-\"6,000\",a,should,https://www.gov.uk/a,1
-"
-      expected = [SearchCheck.new("a", "should", "/a", 1, 6_000)]
-      assert_equal expected, checks(data)
-    end
+    should "skip rows that don't have an integer for the top N number" do
```
Comment (dwhenry) [inline test/unit/health_check/check_file_parser_test.rb:21]: Would be nice to test that this setup error was logged somewhere, but think that is out of scope for this change?

**Review by dwhenry — APPROVED** [review]

File: test/unit/health_check/check_file_parser_test.rb (hunk 1 above)
Comment (MatMoore) [reply test/unit/health_check/check_file_parser_test.rb:21]: Yeah it's not something I've changed.

I think it's logged here: https://github.com/alphagov/rummager/blob/aa4e73f88ef893d379361e0fd3096d793b977252/lib/health_check/check_file_parser.rb#L30
